### PR TITLE
coqprime sync with primo 4.3.1

### PR DESCRIPTION
**Pull request by thery**

### CODE REVIEW

File: released/packages/coq-coqprime/coq-coqprime.1.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "thery@sophia.inria.fr"
+homepage: "http://coqprime.gforge.inria.fr/"
+bug-reports: "http://coqprime.gforge.inria.fr/"
+dev-repo: "git+https://github.com/thery/coqprime.git"
+license: "LGPL"
```
Comment (palmskog) [inline released/packages/coq-coqprime/coq-coqprime.1.0.5/opam]:
```suggestion
license: "LGPL-2.1-only"
```
Can the license be a bit more precise? Based on the repo, this is what I could deduce.

File: released/packages/coq-coqprime/coq-coqprime.1.0.5/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+authors: ["Laurent Théry"]
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Coqprime"]
```
Comment (palmskog) [inline released/packages/coq-coqprime/coq-coqprime.1.0.5/opam]:
```suggestion
```
This line is not needed in OPAM 2.

File: released/packages/coq-coqprime/coq-coqprime.1.0.5/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+depends: [
+  "ocaml"
+  "coq" {>= "8.10~"}
+  "coq-bignums"
+]
+synopsis: "Certifying prime numbers in Coq"
+flags: light-uninstall
```
Comment (palmskog) [inline released/packages/coq-coqprime/coq-coqprime.1.0.5/opam]:
```suggestion
```
This line is not needed in OPAM 2.

File: released/packages/coq-coqprime/coq-coqprime.1.0.5/opam [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+url {
+  src: "https://github.com/thery/coqprime/archive/v8.10b.zip"
```
Comment (palmskog) [inline released/packages/coq-coqprime/coq-coqprime.1.0.5/opam]:
```suggestion
  src: "https://github.com/thery/coqprime/archive/v8.10b.zip"
    checksum: "sha512=6ed573d24cd58fb79170e900d497827304726b67b37fa0d344243b5d7ca9bcfe9effaa193ae5eb2a407344e55032447812d20e27ade0e81efa9c43a3f96b5a0e"
```

Comment (thery) [reply released/packages/coq-coqprime/coq-coqprime.1.0.5/opam]: Thanks

**Pull request merged**